FFXIVClientStructs/FFXIV/Client/LayoutEngine/LayoutWorld.cs

```diff
@@ -18,12 +19,18 @@ public unsafe partial struct LayoutWorld {
     [FieldOffset(0x018)] public LayoutManager* GlobalLayout;
     [FieldOffset(0x020)] public LayoutManager* ActiveLayout;
     //[FieldOffset(0x028)] public LayoutManager* UnkLayout28;
-    //[FieldOffset(0x030)] public LayoutManager* UnkLayout30;
+    [FieldOffset(0x030)] public LayoutManager* PrefetchLayout;
```

Comment:
If this contains the pointer to the prefetched layout, it should be named as such to not have a collision in IDA